Makefile

```diff
@@ -65,3 +65,13 @@ deb: build
 		--description "Boulder is an ACME-compatible X.509 Certificate Authority" \
 		--maintainer "$(MAINTAINER)" \
 		test/config/ sa/_db data/ $(OBJECTS) bin/ct-test-srv
+
+tar: build
+	fpm -f -s dir -t tar --name "boulder" \
+		--license "Mozilla Public License v2.0" --vendor "ISRG" \
+		--url "https://github.com/letsencrypt/boulder" --prefix=/opt/boulder \
+		--version "$(VERSION)" --iteration "$(COMMIT_ID)" --epoch "$(EPOCH)" \
+		--package "$(ARCHIVEDIR)/boulder-$(VERSION)-$(COMMIT_ID).amd64.tar.gz" \
+		--description "Boulder is an ACME-compatible X.509 Certificate Authority" \
+		--maintainer "$(MAINTAINER)" \
+		test/config/ sa/_db data/ $(OBJECTS)
```

Comment:
```suggestion
		test/config/ sa/_db data/ $(OBJECTS)

```

Reply:
I believe that the only flags being used here are:

```

--prefix
--version
--iteration

Filename:
```
boulder-1.0.0-455cdade.amd64.tar.gz
```

```
$ tree opt
opt
└── boulder
    ├── bin
    │   ├── admin-revoker
    │   ├── akamai-purger
    │   ├── bad-key-revoker
    │   ├── boulder
    │   ├── boulder-ca
    │   ├── boulder-observer
    │   ├── boulder-publisher
    │   ├── boulder-ra
    │   ├── boulder-sa
    │   ├── boulder-va
    │   ├── boulder-wfe2
    │   ├── caa-log-checker
    │   ├── ceremony
    │   ├── cert-checker
    │   ├── contact-auditor
    │   ├── crl-storer
    │   ├── crl-updater
    │   ├── expiration-mailer
    │   ├── id-exporter
    │   ├── log-validator
    │   ├── nonce-service
    │   ├── notify-mailer
    │   ├── ocsp-responder
    │   ├── ocsp-updater
    │   ├── orphan-finder
    │   ├── reversed-hostname-checker
    │   └── rocsp-tool
    ├── data
    │   ├── production-email.template
    │   └── staging-email.template
    ├── sa
    │   └── _db
    │       ├── dbconf.yml
    │       └── migrations
    │           ├── 20210223140000_CombinedSchema.sql
    │           ├── 20210308140000_SimplePartitioning.sql
    │           └── 20220328100000_Incidents.sql
    └── test
        └── config
            ├── admin-revoker.json
            ├── akamai-purger.json
            ├── bad-key-revoker.json
            ├── ca-a.json
            ├── ca-b.json
            ├── cert-checker.json
            ├── contact-auditor.json
            ├── contact-exporter.json
            ├── crl-storer.json
            ├── crl-updater.json
            ├── ecdsaAllowList.yml
            ├── expiration-mailer.gotmpl
            ├── expiration-mailer.json
            ├── health-checker.json
            ├── log-validator.json
            ├── nonce.json
            ├── notify-mailer.json
            ├── observer.yml
            ├── ocsp-responder.json
            ├── ocsp-updater.json
            ├── orphan-finder.json
            ├── publisher.json
            ├── ra.json
            ├── rocsp-tool.json
            ├── sa.json
            ├── va-remote-a.json
            ├── va-remote-b.json
            ├── va.json
            └── wfe2.json

8 directories, 62 files
```